### ISSUE TITLE
LT - no headset, dreamdeck runs only without steam vr on Vive

### ISSUE THREAD

**Issue by angrysl0th** [LibreVR/Revive#12]
Hi,

At first I had no entitlement in LT and  in dreamdeck error: unknown source. I had Unknown Sources enabled in oculus and rebooted few times with enable on/off.

Reinstalled oculus (uninstall, install). Still same issue but oculus did not ask me for user name so somewhere data was left(news from previous game installs were also there). So I uninstalled it again and wiped everything manually including regedit and appdata. Installed it again and oculus asked me for my username but unknown sources was already enabled (dono where data about it could be stored).

After this dreamdeck runs in vive without steam vr on or even steam on... Similar how you can run virtual desktop or elite dangerous. When I have steam vr on it does run but does not show up on vive. LT shows no headset error now.

No idea why it works for others just after reboot but not for me. 

Windows 10. GF980.

**Comment by angrysl0th** [comment]
I got something like this but I do not know how to fix it: http://imgur.com/DfMdmNz

Dreamdeck, Henry, Warthunder works - untrusted sources are checked.

**Comment by Rorrim77** [comment]
Did you copy the games to a different location?
Cause when I tried to copy LT I got the same error. But it runs fine on the original location with the patch.

**Comment by LibreVR** [comment]
Do not use the Revive Injector for Lucky's Tale, it has a separate patch: https://github.com/LibreVR/Revive/releases/download/0.2.1/ReviveLT.zip

**Comment by Kaesemuffin** [comment]
got the same issue since yesterday :( Well never tried b4 but I guess there is some invis patch or even a blockade within oculus not to allow "non trusted sources" currently.
Because we both got the symptoms for "not enabling this option"

**Comment by LibreVR** [comment]
I don't think Oculus has patched anything yet, it's just that this is still alpha software so it can break in unknown ways on some setups.

**Comment by Kaesemuffin** [comment]
are you using steam vr beta or even a specific version?
or a specific nvidia / amd driver?

Pls don't mind me asking ... but also I got this problem with the minecraft project as well
last week it worked like a charm since yesterday I've got a grey screen ... no graphics loaded

**Comment by stefanengelen** [comment]
I just did a fresh download of DreamDeck - Unknown sources checked. Revive Injector says succesfully injected dll's. Then a popup appears: Cannot run this application without an Oculus Rift. Patched?

Lucky's tale (downloaded earlier) still works with LT-Riftless.

**Comment by angrysl0th** [comment]
I had all free aps from 30 march purchased in Oculus store. Tested Rvive and LT as soon as it was on reddit. Just my wild guess looking on yellow text in screenshot it seems that Oculus does not recognize it own store game with this hack but still try to check it authenticity as in red text. Rvive is in folder as I tested also LT patch with it - same issue nothing different. 

Game like warthunder works with no issue with rvive so unknown sources is working as it should. Henry, dreamdeck also but I don't need to have steamvr open to run it and when it is on games do not show on on vive screen. This is also strange. In same situation LT shows no headset error and with steam vr on like in picture - headtracking works and I can see this little annoying bug in VIVE. Just WTF. I have no idea why it works different for others just after reboot. As I understand they have to open steamvr first for all games not couting LT error I have. 

Kaesemuffin:
As for Minecraft I have minecrift mod for vive and it works great. Did not test it with rvive. https://www.reddit.com/r/Vive/comments/4degp4/how_to_install_minecrift_vive_minecraft_mod_for/

**Comment by angrysl0th** [comment]
Just to compare what happens in oculus runtime when I:
run dreamdeck http://imgur.com/4Uv1g7q
run LT without steam vr on: http://imgur.com/Tn9EYvq (dramdeck runs without steamvr on)
run LT with steam vr on: http://imgur.com/8VzLNWB (head tracking works + bug is in vive)

Now I officially run out of ideas other than:
Looking on yellow text in screenshot it seems that Oculus does not recognize it own store game with this LT patch but still try to check it authenticity as in red text.

**Comment by angrysl0th** [comment]
Can be closed. I did try again today and same error. Checked LT to run as administrator then played with unknown sources switch. Tried with administrator on "no headset" than without administrator and in run.
Seems that unknown sources check may fail in some cases with LT when LT is not "set run as admin" during change to allow unknown sources. Solution thanks to this guy on reddit.

evesneweden 1 point 23 hours ago 
Yes. I did a few tests prior to doing the unknown sources and reboot and changed it to "run as admin". I disabled that AFTER i enabled unknown sources and rebooted. Successfully started.
